core/src/wallet/cosmos/explorers/GaiaCosmosLikeBlockchainExplorer.cpp

```diff
@@ -427,5 +433,356 @@ namespace ledger {
                     });
         }
 
+        namespace {
+
+        // TODO: probably move most of these in TransactionApi
```

Comment:
If we move them, I'm for creating a new folder altogether just for json serialization/deserialization. The code is in too many places now in my opinion :)

Reply:
Maybe a `GaiaJsonLayer` or something ?

Reply:
I agree, we should gather all these things in the same place, it will probably be easier to use

Reply:
We'll do that after getting send/receive/gas working I suppose